rate/static/html/scale_item.html

```diff
@@ -0,0 +1,5 @@
+<label title="{level}">
```

Comment:
I think a label should always be linked to the associated input element using "for" and an ID tag (like in rate.html above).